modules/apps/asset/asset-categories-admin-web/npmscripts.config.js

```diff
@@ -13,14 +13,25 @@
  */
 
 module.exports = {
+	build: {
+		bundler: {
+			config: {
+				imports: {
+					'asset-taglib': {
+						'/': '>=1.0.0',
+					},
+				},
+			},
+		},
+		dependencies: ['asset-taglib'],
+	},
 	federation: {
```

Comment:
Could't we just remove the whole `federation` entry?

Reply:
What's the benefit of keeping it around?

Reply:
Yes, but that is not causing any error because it is inheriting `mode: disabled` from the base settings. I'll remove it.

The most probable cause is what you say above: some place where we used a different preset. I tried to make sure that all of those cases were left like before, but I probably missed that one.